### ISSUE TITLE
List box SelectionMode="Multiple" not working

### ISSUE THREAD

**Issue by deepaksm1** [wsick/Fayde#177]
List box SelectionMode="Multiple" not working. Checked in Firefox and IE.

```
  <ListBox x:Name="MyListBox" Width="200" Height="100" HorizontalAlignment="Center" VerticalAlignment="Center" SelectionMode="Multiple">
            <ListBoxItem Content="Text 1" />
            <ListBoxItem Content="Text 2" />
            <ListBoxItem Content="Text 3" />
            <ListBoxItem Content="Text 4" />
            <ListBoxItem Content="Text 5" />
            <ListBoxItem Content="Text 6" />
            <ListBoxItem Content="Text 7" />
            <ListBoxItem Content="Text 8" />
            <ListBoxItem Content="Text 9" />
            <ListBoxItem Content="Text 10" />
        </ListBox>
```

**Comment by BSick7** [comment]
Selection is working fine, but the visuals aren't reflecting.  The default theme for ListBoxItem has a blank visual state for `SelectedUnfocused`.  When an item went out of focus, the style would clear any visuals.  Duplicating `Selected` visual state for `SelectedUnfocused`.

**Comment by BSick7** [comment]
Fixed in v0.16.56.
